src/core/print_ps.cpp

```diff
@@ -564,7 +564,7 @@ void PrintShowToPS::PrintContSections(std::ostream& buffer, const CC_sheet& shee
 	for (auto& text : continuity)
 	{
 		if (!text.GetOnSheet()) continue;
-		gen_cont_line(buffer, text, PSFONT_NORM, this_size);
+		gen_cont_line(buffer, text, PSFONT_TYPE::NORM, this_size);
```

Comment:
Just wondering, why are PSFONT_TYPE and SYMBOL_TYPE uppercase snake case while everything else seems to me capital camel case?